src/pages/HttpBin/index.js

```diff
@@ -48,30 +47,25 @@ const HttpBin = () => {
   const createRequest = () => {
     return new Promise((resolve, reject) => {
       const baseURL = 'https://httpbin.org/';
-      const url =
-        urlFormValue.length === 0 ? methodType.toLowerCase() : urlFormValue;
-
-      try {
-        const axiosConfig = {
-          method: methodType,
-          baseURL: baseURL,
-          data: bodyParameters,
-          url: url,
-          params: JSON.parse(queryParameters),
-        };
-
-        setTimeout(() => {
-          axios(axiosConfig)
+      setTimeout(() => {
+        try {
+          axios({
+            method: methodType,
+            baseURL: baseURL,
+            data: bodyParameters,
+            url: methodType.toLowerCase(),
+            params: JSON.parse(queryParameters),
+          })
             .then(({ data }) => {
               resolve(data);
             })
             .catch((error) => {
               reject(error);
```

Comment:
could you send a screenshot from this reject?
I wanna see a screenshot that appears after this reject call

Reply:
![Screenshot_1](https://user-images.githubusercontent.com/17178553/93114066-c6f11c80-f6c2-11ea-8a51-6489d7255f62.png)
this error handled axios